network_importer/adapters/netbox_api/adapter.py

```diff
@@ -64,6 +63,39 @@ class NetBoxAPIAdapter(BaseAdapter):
 
     query_device_info_from_netbox = query_device_info_from_netbox
 
+    @staticmethod
+    def _is_tag_present(netbox_obj):
+        """Find if tag is present for a given object."""
+        if isinstance(netbox_obj, dict) and netbox_obj.get("tags", None):  # pylint: disable=no-else-return
+            return False
```

Comment:
This seems strange - if it's a dict and has a `"tags"` key, we return False immediately? Are we missing a `not` or two here? I would have expected `if isinstance(netbox_obj, dict) and "tags" not in netbox_obj` or something like that instead.

Reply:
My thinking was if there is no tag key then we should fail (return) early, as there is nothing more we need to do regarding these flag functions.

Reply:
Fixed.